web/src/components/tasks/RaffleBingoPaginator.tsx

```diff
@@ -0,0 +1,193 @@
+        raffleBingoStepFiles.map(async (fileName) => {
+          const step = await fetchTaskByFilename(fileName);
+          return { name: step.stepLabel ?? "" };
+        })
+      );
+      setStepLabels(labels);
+    };
+    getStepLabels();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const onMoveToStep = (stepIndex: number): void => {
+    setStepIndex(stepIndex);
+  };
+
+  const getContinueButtonText = (stepIndex: number): string => {
+    if (stepIndex === raffleBingoStepFiles.length - 1) {
+      return Config.taskDefaults.backButtonText;
+    } else {
+      return Config.taskDefaults.continueButtonText;
+    }
+  };
+
+  const markTaskAsComplete = (): void => {
+    if (!business || !updateQueue) return;
+    const completed: TaskProgress = "COMPLETED";
+    updateQueue.queueTaskProgress({ [props.task.id]: completed });
+    updateQueue.update();
+    setSuccessSnackbarIsOpen(true);
+  };
+
+  const displayCompletedSnackbar = (): ReactNode => {
+    const currentTaskProgress: TaskProgress = business?.taskProgress[props.task?.id] ?? "COMPLETED";
+
+    return (
+      <TaskStatusChangeSnackbar
+        isOpen={successSnackbarIsOpen}
+        close={(): void => setSuccessSnackbarIsOpen(false)}
+        status={currentTaskProgress}
+      />
+    );
+  };
+
+  const stepItems: StepperStep[] = stepLabels.map((item) => {
+    return { name: item.name ?? "" };
+  });
+
+  const agencyName = step?.agencyId ? LookupTaskAgencyById(step.agencyId).name : "";
+
+  return (
+    <>
+      <TaskHeader task={props.task} />
+      <Content>{step?.summaryDescriptionMd ?? ""}</Content>
+      {stepItems.length > 0 ? (
+        <HorizontalStepper
+          steps={stepItems}
+          currentStep={stepIndex}
+          onStepClicked={(step: number): void => {
+            onMoveToStep(step);
+          }}
+        />
+      ) : (
+        <div>Loading...</div>
+      )}
+      <div data-testid={step?.filename}>
+        <Content>{step?.contentMd ?? ""}</Content>
+        <div className="margin-bottom-4 margin-left-2">
+          <SingleCtaLink
+            link={step?.callToActionLink ?? ""}
+            text={step?.callToActionText}
+            buttonColor="outline"
+            noBackgroundColor={true}
+            alignLeft={true}
+            iconName="launch"
+          />
+        </div>
+      </div>
+      <HorizontalLine />
+      <div>
+        <span className="text-bold">Issuing Agency: </span>
+        {agencyName}
+      </div>
+
+      <div>
+        <CtaContainer noBackgroundColor={false}>
+          <ActionBarLayout>
+            {isNotLastStep(stepIndex) && (
+              <PrimaryButton
+                isColor="primary"
+                onClick={(): void => {
+                  onMoveToStep(stepIndex + 1);
+                }}
+                dataTestId="continue-button"
+                isRightMarginRemoved={true}
+              >
+                {getContinueButtonText(stepIndex)}
```

Comment:
[sand] The way you have this logic written now, I don't think you actually need a function `getContinueButtontext`. I also don't think the Back button logic is actually getting executed in that function.
```suggestion
                {Config.taskDefaults.continueButtonText}
```